includes/class-freemius.php

```diff
@@ -509,23 +547,69 @@ private function _register_account_hooks() {
 		 * @uses   fs_find_caller_plugin_file
 		 */
 		private function _find_caller_plugin_file() {
-			// Try to load the cached value of the file path.
+			return $this->_storage->plugin_main_file->path;
+		}
+
+
+		/**
+		 * Identifies the caller type: plugin or theme.
+		 *
+		 * @author Leo Fajardo (@leorw)
+		 * @since  1.2.0
+		 *
+		 * @return string
```

Comment:
@fajardoleo the method calculates that, but it doesn't actually return anything. So `@return string` should be removed from the function documentation.